pkg/frontend/admin_openshiftcluster_vm.go

```diff
@@ -0,0 +1,63 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database/cosmosdb"
+)
+
+const (
+	redeployVM = iota
+	restartVM
+	startVM
+)
+
+func (f *frontend) _postAdminOpenShiftClusterVM(ctx context.Context, r *http.Request, log *logrus.Entry, vmAction int) error {
```

Comment:
Since this frontend method is in its own file now we should rename it to remove the `_` prefix. In addition we can make the name a bit clearer on its intent. What do you think about `postAdminOpenShiftClusterVMAction`?